exo/inference/pytorch/test_inference_engine.py

```diff
@@ -0,0 +1,33 @@
+
+import asyncio
+from exo.inference.shard import Shard
+from exo.inference.pytorch.inference import PyTorchDynamicShardInferenceEngine
+
+def main():
+    shard = Shard(
+        model_id="meta-llama/Meta-Llama-3.1-8B",
+        start_layer=0,
+        end_layer=0,
+        n_layers=12
+    )
+
+    engine = PyTorchDynamicShardInferenceEngine(
```

Comment:
Is this test complete? We need a test that tests the model splitting. Take a look at `exo/inference/test_inference_engine.py`. You can just add the test there.